frontend/src/queries/nodes/InsightQuery/utils/filtersToQueryNode.ts

```diff
@@ -296,22 +289,22 @@ export const filtersToQueryNode = (filters: Partial<FilterType>): InsightQueryNo
 
     // breakdown
     if (isInsightQueryWithBreakdown(query)) {
-        /* handle multi-breakdowns */
+        const isLegacyMultipleBreakdown = filters.breakdowns?.find((b) => 'property' in b)
+
+        /* handle legacy multi-breakdowns */
         // not undefined or null
-        if (filters.breakdowns != null) {
-            if (filters.breakdowns.length === 1) {
-                filters.breakdown_type = filters.breakdowns[0].type
-                filters.breakdown = filters.breakdowns[0].property as string
-            } else {
-                captureException(
-                    'Could not convert multi-breakdown property `breakdowns` - found more than one breakdown'
-                )
-            }
+        if (filters.breakdowns != null && isLegacyMultipleBreakdown) {
+            filters.breakdowns = filters.breakdowns
```

Comment:
I'm worrying that this will break compatibility with the multi-breakdown format in funnels. Not sure what the state of that is concretely, but at least the syntax is used quite frequently. See [this question](https://metabase.prod-us.posthog.dev/question/869-multi-breakdown-usage) in metabase.

Now I realize this feedback comes quite late (and likely too late), but is there a specific reason we need to change from `property` to `value`? Given that we use `property` for filters as well and "breakdown values" usually refers to the concrete values e.g. "Chrome", "Safari", I'd think `property` is more consistent.

Reply:
There is no specific reason. We can still change the field back to `property` because there are only four saved insights with the `value` field, which are all from internal users. The arguments to change the field to `property` are reasonable. I didn't know that we have many saved insights with multiple breakdowns.

Reply:
Ok cool! Might be more reasonable to make this change then. I think we have the saved insights because the PostHog team is using a feature-flagged version of  funnel multi-breakdowns and insights generated with this feature flag made it into templates.